sktime/forecasting/trend/_stl_forecaster.py

```diff
@@ -138,7 +139,7 @@ class STLForecaster(BaseForecaster):
     """
 
     _tags = {
-        "authors": ["tensorflow-as-tf", "mloning", "aiwalter", "fkiraly"],
+        "authors": ["tensorflow-as-tf", "mloning", "aiwalter", "fkiraly","ericjb"],
```

Comment:
formatting failure - there should be a space

Here's how you can set up the checks locally, to avoid that you need to wait for the failure on the remote CI which may take some time to start: https://www.sktime.net/en/latest/developer_guide/coding_standards.html#setting-up-local-code-quality-checks